packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -263,6 +263,51 @@ describe('<Tabs />', () => {
         expect(style.left).to.equal('60px');
         expect(style.width).to.equal('50px');
       });
+
+      it('should have "right" for RLT', () => {
```

Comment:
```suggestion
      it('should have "right" for RTL', () => {
```

these typos can be very problematic when searching the codebase